docs/source/pages/tutorials/exclude_include_checks.rst

```diff
@@ -0,0 +1,174 @@
+.. Copyright (c) 2024 - 2024, Oracle and/or its affiliates. All rights reserved.
+.. Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
+
+=====================================
+Exclude and include checks in Macaron
+=====================================
+
+.. note::
+
+    This feature is only available from Macaron 0.8.0
+
+This tutorial will show how you can configure Macaron to:
+
+* exclude checks that you don't want to run.
+* include only checks that you want to run.
+
+.. note::
+
+    By default, Macaron will run all of its checks.
+
+-------------
+Prerequisites
+-------------
+
+* You are expected to have gone through :ref:`this tutorial <detect-malicious-java-dep>`.
+* This tutorial requires a high-level understanding of checks in Macaron and how they depend on each other. Please see this :ref:`page <macaron-developer-guide>` for more information.
+
+------------------
+Motivating example
+------------------
+
+Let's say you want to run the analysis on `the micronaut-projects/micronaut-core Github repository <https://github.com/micronaut-projects/micronaut-core>`_ at the tag ``4.3.10``.
+
+Normally, this is how you would run Macaron:
+
+.. code-block:: shell
+
+  ./run_macaron.sh analyze --package-url pkg:github.com/micronaut-projects/micronaut-core@v4.3.10 --skip-deps
+
+However, there can be checks in Macaron that are not applicable to ``micronaut-projects/micronaut-core``.
+For example, the ``mcn_provenance_witness_level_one_1`` check (defined in :class:`ProvenanceWitnessL1Check <macaron.slsa_analyzer.checks.provenance_witness_l1_check.ProvenanceWitnessL1Check>`) is not applicable because ``micronaut-projects/micronaut-core`` doesn't generate and publish any `Witness provenance <https://github.com/testifysec/witness>`_. Therefore, we could exclude this check from running by performing the following steps.
+
+1. In your current directory, create a ``defaults.ini`` configuration file with the following content.
+
+.. code-block:: ini
+
+    [analysis.checks]
+    exclude = mcn_provenance_witness_level_one_1
+    include = *
+
+In the ini configuration above:
+
+* The ``exclude`` option contains the check ID ``mcn_provenance_witness_level_one_1``. The check with this ID will be excluded.
+* The ``include`` option contains the `glob <https://docs.python.org/3/library/glob.html>`_ pattern ``*``. Checks whose ID matched this glob pattern will be included in the analysis. In this case, ``*`` matches all check IDs.
```

Comment:
```suggestion
* The ``include`` option contains the `glob <https://docs.python.org/3/library/glob.html>`_ pattern ``*``. Checks with IDs matching this glob pattern will be included in the analysis. In this case, ``*`` matches all check IDs.
```

Reply:
Fixed in https://github.com/oracle/macaron/pull/667/commits/2389ea38eb37dc52a06e7c1b0e15726fb89a6a10